src/OpenFeature/Constant/ErrorType.cs

```diff
@@ -16,40 +14,40 @@ public enum ErrorType
     /// <summary>
     /// Provider has yet been initialized
     /// </summary>
-    [Description("PROVIDER_NOT_READY")] ProviderNotReady,

```

Comment:
Think this may be a breaking change if the client tries to get the description from `DescriptionAttribute`

Reply:
I tried to remove it because we are now using the switch, but I see your point. The consumer might still be using the `DescriptionAttribute`.

Reply:
Reverted with: https://github.com/open-feature/dotnet-sdk/pull/554/commits/92339a642e04de6e78b0350523de18d06d582cb2